### PR TITLE
Point corporate donors toward our OC contact form

### PR DESCRIPTION
**Pull request by colindean**
A corporate donor recently wasn't able to find a clear way to contact a Homebrew person to talk about a possible donation. They eventually found the OC contact us page, which is the best way to contact the PLC short without having a GitHub account and posting on an issue or Discussions.

Let's highlight the donations questions contact form specifically in this area that is the landing target for the donations link on https://brew.sh.

---

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] ~Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).~
- [ ] ~Have you successfully run `brew style` with your changes locally?~
- [ ] ~Have you successfully run `brew typecheck` with your changes locally?~
- [ ] ~Have you successfully run `brew tests` with your changes locally?~

-----

### CODE REVIEW

**Review by Bo98 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Homebrew is a non-profit project run entirely by unpaid volunteers. We need your
 
 Please consider a regular donation through [GitHub Sponsors](https://github.com/sponsors/Homebrew), [Open Collective](https://opencollective.com/homebrew) or [Patreon](https://www.patreon.com/homebrew). Homebrew is fiscally hosted by the [Open Source Collective](https://opencollective.com/opensource).
 
+For questions about donations, including corporate giving, please contact us through our Open Collective [contact form](https://opencollective.com/homebrew/contact).
```
Comment (MikeMcQuaid) [inline README.md:30]: Can we use plc@brew.sh, billing@brew.sh, a new address or similar instead here? The OpenCollective form is not great (as demonstrated by the fact we replied to this multiple times without knowing this time)?

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (colindean) [reply README.md:30]: Certainly.

```suggestion
For questions about donations, including corporate giving, please email the Homebrew PLC at [plc@brew.sh](mailto:plc@brew.sh) or contact us through our Open Collective [contact form](https://opencollective.com/homebrew/contact).
```

Comment (MikeMcQuaid) [reply README.md:30]:
```suggestion
For questions about donations, including corporate giving, email the Homebrew PLC at [plc@brew.sh](mailto:plc@brew.sh).
```

**Review by MikeMcQuaid — APPROVED** [review]
Thanks @colindean!

**Pull request merged**